lib/src/date-input/DateInput.tsx

```diff
@@ -278,46 +151,37 @@ const DxcDateInput = React.forwardRef<RefType, DateInputPropsType>(
                 margin={margin}
                 size={size}
                 tabIndex={tabIndex}
-                ref={refDate}
+                ref={dateRef}
               />
-              <Popover
-                onKeyDown={handleCalendarOnKeyDown}
-                open={isOpen}
-                anchorEl={anchorEl}
-                anchorOrigin={{
-                  vertical: "bottom",
-                  horizontal: "left",
-                }}
-                transformOrigin={{
-                  vertical: "top",
-                  horizontal: "center",
-                }}
-                PaperProps={{
-                  style: {
-                    marginTop: "10px",
-                  },
-                }}
-              >
-                <ClickAwayListener onClickAway={closeCalendar}>
-                  <Paper role="dialog" aria-modal="true">
-                    <DatePicker
-                      variant="static"
-                      value={getValueForPicker(value ?? innerValue, format)}
-                      onChange={(date) => handleCalendarOnClick(date)}
-                      format={format}
-                      disabled={disabled}
-                    />
-                  </Paper>
-                </ClickAwayListener>
-              </Popover>
-            </StyledDPicker>
-          </MuiPickersUtilsProvider>
-        </MuiThemeProvider>
+            </Popover.Trigger>
+            <StyledContent
+              sideOffset={error ? -18 : 2}
+              align="end"
+              aria-modal={true}
+              onFocusOutside={handleFocusOutside}
+              avoidCollisions={false}
+            >
+              <DxcDatePicker
+                id={calendarId}
+                onCloseCalendar={closeCalendar}
+                onEscCalendar={handleEscCalendar}
+                onDateSelect={handleCalendarOnClick}
+                date={getValueForPicker(value ?? innerValue, format.toUpperCase())}
+              />
+            </StyledContent>
+          </Popover.Root>
+        </DxcDateInputContainer>
       </ThemeProvider>
     );
   }
 );
 
-const StyledDPicker = styled.div``;
+const StyledContent = styled(Popover.Content)`
+  &:focus-visible {
+    outline: none;
+  }
+`;
+
+const DxcDateInputContainer = styled.div``;
```

Comment:
The excess of styled components can affect performance, we should remove this and use directly `div`.